testerum-backend/testerum-runner/testerum-runner-cmdline/src/main/kotlin/com/testerum/runner_cmdline/RunnerApplication.kt

```diff
@@ -56,14 +61,21 @@ class RunnerApplication(private val runnerProjectManager: RunnerProjectManager,
         }
     }
 
-    private fun tryToExecute(cmdlineParams: CmdlineParams): ExitCode {
+    fun getRunnerSuiteToBeExecuted(cmdlineParams: CmdlineParams): RunnerSuite {
         initialize(cmdlineParams)
 
-        // create execution tree
         val testsDir = runnerProjectManager.getProjectServices().dirs().getTestsDir()
-        val suite: RunnerSuite = runnerExecutionTreeBuilder.createTree(cmdlineParams, testsDir)
-        logRunnerSuite(suite)
+        val suite: RunnerSuite = runnerExecutionTreeBuilder.createTree(cmdlineParams, testsDir);
+        return suite;
```

Comment:
Remove semicolons + enter before return:
```suggestion
        val suite: RunnerSuite = runnerExecutionTreeBuilder.createTree(cmdlineParams, testsDir)

        return suite
```